src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends Ball {
+
+    public Ball getRandomBall() {
+
+        Ball ball = new Ball();
+
+        ball.setColor(new ColorSupplier().getRandomColor());
+        ball.setNumber(new Random().nextInt(100));
```

Comment:
All magic numbers in your code should be constants. The number '100' should be extracted into a constant with a meaningful name.